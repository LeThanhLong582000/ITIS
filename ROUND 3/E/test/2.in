93
81 1
1 80
93 21
1 1
61 61
61 45
41 45
49 1
23 37
19 87
65 13
67 89
1 51
97 61
9 21
21 100
69 1
21 35
33 81
96 81
65 17
37 60
7 31
81 51
49 1
21 33
1 1
55 47
53 46
9 61
1 7
1 33
37 55
80 53
81 61
65 52
81 53
79 76
38 57
51 66
1 89
91 71
1 33
61 39
13 32
53 61
65 95
83 1
57 71
61 81
1 36
11 55
91 26
33 14
1 1
81 37
88 89
11 81
1 66
11 49
99 25
76 26
67 33
41 51
65 5
53 1
88 51
1 41
28 76
97 79
1 1
21 13
73 76
53 85
21 41
11 29
1 81
70 21
41 49
29 67
41 1
21 71
69 21
89 6
41 1
13 1
49 49
3 1
17 39
77 21
1 65
3 1
32 39
