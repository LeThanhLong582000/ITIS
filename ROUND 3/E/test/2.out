6480
6320
3797984
0
13813800
7511576
3388436
2352
717246
2718788
707340
35505558
2550
34959516
34364
4392536
4692
534266
7122224
60398300
1212048
4909568
45576
17029550
2352
474644
0
6660168
5923074
297284
42
1056
4123998
17941020
24370700
11395368
18392804
35995824
4673186
11301050
7832
41688300
1056
5639414
169836
10424544
38076954
6806
16343306
24370700
1260
361356
5578050
209834
0
8956436
61271844
786950
4290
286374
6104822
3888000
4869806
4354550
103524
2756
20103404
1640
4510464
58654698
0
72464
30732144
20256120
734300
99354
6480
2148746
4019064
3758898
1640
2210750
2087624
281474
1640
156
5744352
6
434258
2601296
4160
6
1547108
